code/src/main/java/frc/robot/RobotConfig.java

```diff
@@ -42,6 +44,8 @@
         public final int rearRightAbsoluteEncoder = 2;
         public final double swerveX = .67 / 2;
         public final double swerveY = .25;
+        public final SwerveDriveKinematics kinematics = new SwerveDriveKinematics(new Translation2d(swerveX, swerveY), new Translation2d(swerveX, -swerveY),
```

Comment:
any reason for not just having this code in the SwerveDrivetrain class? It doesn't seem to be used anywhere else and this isn't  a constant in terms of a simple value.